agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/DiseaseAnnotationCurationIndexer.java

```diff
@@ -75,13 +75,15 @@ protected void index() {
 		diseaseRepository = new DiseaseRepository();
 		closureMap = diseaseRepository.getDOClosureChildMapping();
 
-		indexGenes();
+		//indexGenes();
```

Comment:
@cmpich do you really want to turn off indexing genes?

Reply:
Good catch. Undid it.